app/scripts/presenters/visit_presenter.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
On the topic of filenames... I noticed some recent refactorings from underscores to dashes, not sure if we have a coding style guide yet or if it's worth worrying about early, or if we want to change these to "visit-presenter.js" to match "/server/work-queue/jobs/extract-page.js". /cc @6a68

Possibly same question applies to filenames like "app/images/icon-favicon_default@2x.png" above (dashes and underscores). I can try doing some shell-fu to see if I can find the incantation for listing all filenames minus special folders like .git, node_modules, app/bower_components, etc...